### DISCUSSION TITLE
Release date is showing as "unreleased" in curl 8.0.1

### DISCUSSION THREAD

**Discussion by padmeshgit** [curl/curl#10824] [Q&A] [answered]
I have built a curl 8.0.1 and after running "curl --version" command it is not showing release date.

`curl 8.0.1-DEV (x86_64-pc-linux-gnu) libcurl/8.0.1-DEV OpenSSL/1.1.1k-fips zlib/1.2.11 brotli/1.0.6 zstd/1.4.4 libidn2/2.2.0 nghttp2/1.33.0
**Release-Date: [unreleased]**
Protocols: dict file ftp ftps gopher gophers http https imap imaps mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli HSTS HTTP2 HTTPS-proxy IDN IPv6 Largefile libz NTLM NTLM_WB SSL threadsafe TLS-SRP UnixSockets zstd`

Am i missing something? Please suggest

**Comment by bagder** [comment, marked as answer]
This means that you did not build curl from a release package but probably straight from git? The release date is set automatically in releases. Releases are the dedicated (blessed and signed) tarballs we call releases.

**Reply by padmeshgit** [reply]
@bagder I have built it from release tag only (curl-8_0_1) and I had built the older version (7.86) in the same way where it is showing the release date.

**Reply by bagder** [reply]
No you did not. The release date never shows when you build from a git tag. A git tag is not a curl release. A tag just marks that moment in time the release was generated from.

curl releases are the blessed and signed **tarballs** created and uploaded by me, as releases. They contain the correct version numbers, release dates and and more.

**Reply by padmeshgit** [reply]
Thanks @bagder for your clarification.

Appreciated!
